edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeIdentitiesCache.cs

```diff
@@ -17,27 +17,40 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Core
 
     public sealed class DeviceScopeIdentitiesCache : IDeviceScopeIdentitiesCache
     {
+        static readonly TimeSpan DefaultRefreshDelay = TimeSpan.FromMinutes(5);
```

Comment:
5 mins seems too much. What do we use in 1.2? I would suggest 1 min or 2 mins.

Reply:
For me, this is just to prevent repeated calls to the cloud (say in the span of a second)

Reply:
it's 2 min in 1.2,  updated here to 2 mins